internal/provider/resource_instance.go

```diff
@@ -1020,6 +1061,33 @@ func newAttachedNetworkInterfacesModel(ctx context.Context, client *oxide.Client
 	return nicSet, nil
 }
 
+func newAttachedExternalIPModel(ctx context.Context, client *oxide.Client, instanceID string) (
+	[]instanceResourceExternalIPModel, diag.Diagnostics) {
+	var diags diag.Diagnostics
+	externalIPs := make([]instanceResourceExternalIPModel, 0)
+
+	// TODO: Support pagination when Go SDK supports it.
```

Comment:
The Go SDK supports pagination in all endpoints that have it. In this case, it appears the endpoint doesn't have pagination https://github.com/oxidecomputer/omicron/blob/62614ab4abe2a4bc9976dd0e2fb395bc544413c3/openapi/nexus.json#L3623-L3668

I don't really know if it's in any one's roadmap to support pagination for this endpoint. Let's change the comment (or remove it, there's not much to do on our side tbh 🤷‍♀️ )

Reply:
I removed the comment. The context you provided was great, thank you!